docs/plainmonthday.md

```diff
@@ -20,18 +20,22 @@ A `Temporal.PlainMonthDay` can be converted into a `Temporal.PlainDate` by combi
 - `isoMonth` (number): A month, ranging between 1 and 12 inclusive.
 - `isoDay` (number): A day of the month, ranging between 1 and 31 inclusive.
 - `calendar` (optional `Temporal.Calendar` or plain object): A calendar to project the date into.
-- `referenceISOYear` (optional number): A reference year, used for disambiguation when implementing other calendar systems.
-  The default is the first leap year after the [Unix epoch](https://en.wikipedia.org/wiki/Unix_time).
-  You can omit this parameter unless using a non-ISO-8601 calendar.
+- `referenceISOYear` (optional for ISO 8601 calendar; required for other calendars):
+  A reference year in the ISO 8601 calendar for disambiguation when implementing calendar systems.
+  The default for the ISO 8601 calendar is the first leap year after the [Unix epoch](https://en.wikipedia.org/wiki/Unix_time).
+  This value is required for other calendar systems, but can cause serious problems if not set correctly.
```

Comment:
This might be a bit dramatic — how about this?
```suggestion
  This value is required for other calendar systems. If you don't know what to put here, use the `from` method instead.
```